src/components/taskDetails/index.tsx

```diff
@@ -121,9 +128,41 @@ const TaskDetails: FC<Props> = ({ url, taskID }) => {
             );
         }
     }
-
+    const fetchDependentTasks = async (
+        taskDetails: taskDetailsDataType['taskData']
+    ) => {
+        try {
+            if (taskDetails?.dependsOn) {
+                const dependsOnTitles = await Promise.all(
+                    taskDetails.dependsOn.map(async (taskId: string) => {
+                        const { requestPromise } = fetch({
+                            url: `${TASKS_URL}/${taskId}/details`,
+                        });
+                        const data = await requestPromise;
+                        return [data?.data?.taskData?.title, taskId];
+                    })
+                );
+                const titles = dependsOnTitles.map(
+                    (innerArray) => innerArray[0]
+                );
+                const ids = dependsOnTitles.map(
+                    (innerArrays) => innerArrays[1]
+                );
+                setTaskTitle(titles);
+                setId(ids);
+                setIsFetched(true);
+            }
+        } catch (error) {
+            console.error('Error while fetching taskdependency', error);
```

Comment:
I think toast is not necessary here because task dependency is a particular section, toast is required for the whole task detail to load a particular task